agent-control/src/bin/main_agent_control_onhost_cli.rs

```diff
@@ -20,9 +20,11 @@ struct Cli {
 #[derive(Debug, clap::Subcommand)]
 enum Commands {
     // Generate Agent Control configuration according to the provided configuration data.
-    GenerateConfig(config_gen::Args),
+    AgentConfig(config_gen::Args),
```

Comment:
By changing the name we are also changing the command. I'm totally fine with the change but we need to update the recipe accordingly.

❗ Since we are already testing it in the recipe, we already need a branch in the recipes repository to make the on-host e2e tests pass.

Reply:
I won't change it, I am not sure why I did the change, the name is okey for me.